cranelift/codegen/src/isa/x64/inst.isle

```diff
@@ -1488,6 +1488,15 @@
 
 ;;;; Helpers for Emitting Loads ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+;; Helper for constructing a LoadExtName instruction.
+(decl load_ext_name (ExternalName i64) Reg)
+(rule (load_ext_name extname offset)
+      (let ((dst WritableGpr (temp_writable_gpr))
+            (_ Unit (emit (MInst.LoadExtName dst
+                                             (box_external_name extname)
```

Comment:
would it be worthwhile to have an auto-conversion for this?

Reply:
Definitely! I'll add that in.